ydb/core/tx/replication/controller/tx_resolve_database_result.cpp

```diff
@@ -0,0 +1,79 @@
+#include "controller_impl.h"
+
+namespace NKikimr::NReplication::NController {
+
+class TController::TTxResolveDatabaseResult : public TTxBase {
+    const TEvPrivate::TEvResolveTenantResult::TPtr Ev;
+    TReplication::TPtr Replication;
+
+public:
+    explicit TTxResolveDatabaseResult(TController* self, TEvPrivate::TEvResolveTenantResult::TPtr& ev)
+        : TTxBase("TxResolveDatabaseResult", self)
+        , Ev(ev)
+    {
+    }
+
+    TTxType GetTxType() const override {
+        return TXTYPE_RESOLVE_DATABASE_RESULT;
+    }
+
+    bool Execute(TTransactionContext& txc, const TActorContext& ctx) override {
+        CLOG_D(ctx, "Execute: " << Ev->Get()->ToString());
+
+        const auto rid = Ev->Get()->ReplicationId;
+        const auto& tenant = Ev->Get()->Tenant;
+
+        Replication = Self->Find(rid);
+        if (!Replication) {
+            CLOG_W(ctx, "Cannot resolve database of unknown replication"
+                << ": rid# " << rid);
+            return true;
+        }
+
+        if (Ev->Get()->IsSuccess()) {
+            CLOG_N(ctx, "Tenant resolved"
```

Comment:
Везде стало `Database`, а здесь осталось `Tenant`.
```suggestion
            CLOG_N(ctx, "Database resolved"
```